crabs/tracker/track_video.py

```diff
@@ -88,7 +88,7 @@ def load_trained_model(self) -> torch.nn.Module:
             self.args.checkpoint_path
         )
         trained_model.eval()
-        trained_model.to(DEVICE)  # Should device be a CLI?
+        trained_model.to(DEVICE)
         return trained_model
 
     def load_video(self) -> None:
```

Comment:
> In run_tracking() below, we could combine the two if self.args.gt_path cases together.

I forgot to address this. You had some changes like this before in refactoring. But the evaluation will happen only towards the end, when all the frames have been tracked. I added the gt_path checking at the beginning because it can takes a long time to finish the tracking and only then got error that the path is not exist.

Reply:
> n run_tracking()we set frame_number = 1 ...

Shouldn't be an issue, but good call, I just change to 0 to make it consistent

Reply:
> I forgot to address this. You had some changes like this before in refactoring. But the evaluation will happen only towards the end, when all the frames have been tracked. I added the gt_path checking at the beginning because it can takes a long time to finish the tracking and only then got error that the path is not exist.

Ok let's leave it as is, ideally we implement validators at some point - I opened an issue #211